### DISCUSSION TITLE
Blank space before the name in subject search brings more results than no space.

### DISCUSSION THREAD

**Discussion by cvinhaes** [photoprism/photoprism#3926] [Q&A] [answered]
I'm experiencing a weird behavior with the search. Searching for a subject brings a number of results if I type the name of the subject just next to the colon (subject:xxx) and a much higher number if I add an space between the name and the colon (subject: xxx). I have no idea of what may cause this. Running the official Docker image of PhotoPrism® Essentials Build 231128-f48ff16ef on Debian 10.2.1-6, Intel Silicon.

**Comment by graciousgrey** [comment, marked as answer]
If you search for `subject:jane` or `subject:"jane doe"` the subject search filter is used and PhotoPrism only shows photos that are assigned to the person Jane.

https://docs.photoprism.app/user-guide/search/filters/#filter-reference

If you search for `subject: jane` the search filter is not used and PhotoPrism shows all photos it finds related to the search term "jane" those might be photos that are assigned to the person jane but also photos that have the keyword jane.

**Reply by cvinhaes** [reply]
Very clear, thank you so much! One feature suggestion I have provided in the past is to offer a search menu which would help to bypass the idiosyncrasies of the search syntax.

**Reply by graciousgrey** [reply]
You are welcome! A user interface for the search filters is planned, but we can't tell you yet when we'll get to it :)
